docs/technical-setup/environment-setup/setup-conda.md

```diff
@@ -25,47 +25,66 @@ There are two main reasons we use conda for OpenScPCA:
 
 ## Install conda
 
-We recommend installing [Miniconda](https://docs.anaconda.com/free/miniconda/index.html) to obtain conda.
-Miniconda is lightweight version of the full conda platform and includes the conda tool itself, Python, and a few other commonly-used packages.
+We recommend installing [Miniforge](https://github.com/conda-forge/miniforge?tab=readme-ov-file#miniforge) to obtain conda.
+Miniforge is lightweight version of the conda distribution that is free and open source.
+The installation includes the conda tool itself, Python, and a few other commonly-used packages.
 
 If you already have conda on your system, you do not need to re-install it.
 
 
-### Install Miniconda on macOS
+### Installing Miniforge
 
-To install Miniconda, [download the graphical installer for macOS](https://docs.anaconda.com/free/miniconda/miniconda-install/), and follow all instructions.
+To install Miniforge, follow the [installation instructions in the Miniforge repository](https://github.com/conda-forge/miniforge?tab=readme-ov-file#miniforge).
 
-  - If you are on a macOS computer, be sure to download one of the links ending in `pkg`, _not `bash`_:
-    - Apple silicon (M1-series) Mac users should download `Miniconda3 macOS Apple M1 64-bit pkg`
-    - Intel Mac users should download `Miniconda3 macOS Intel x86 64-bit pkg`
+Briefly, you will need to open a terminal and run the following commands:
 
-### Install Miniconda on Windows with WSL 2
+```sh
+curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
+bash Miniforge3-$(uname)-$(uname -m).sh
+```
 
-To install Miniconda onto the WSL 2 side of your Windows machine, you will need to follow the [Quick command line install instructions for Linux](https://docs.anaconda.com/free/miniconda/#quick-command-line-install).
+!!! note
+    On Windows with WSL 2, you will need to install Miniforge on the WSL 2 side of your system, starting from the [Ubuntu terminal](../../getting-started/project-tools/using-the-terminal.md).
 
-- Copy and paste all commands, in order, into the [Ubuntu terminal](../../getting-started/project-tools/using-the-terminal.md).
-  - Be sure to also take the step to initialize conda for the Bash shell.
-- Then, as prompted, close your terminal and open a new terminal window to complete installation.
+You will be prompted first to view the license; press enter as instructed, read the license (you should be able to scroll with your arrow keys), then press `q` to return to the installer.
```

Comment:
A little fiddly here...
When I tried this, I found if you just hit the down arrow to scroll, it ends up prompting you for yes/no and you don't ever need `q`. If otoh, you hit `q` without scrolling, it goes straight to yes/no.

Reply:
Yeah, it can be a bit system dependent... if you hit down after you get to the end it goes back, and actually many other keys will take you back (space, for example). You can also hit return repeatedly to scroll. I just went with what I thought was likely to be most universal when you get dumped into a pager like that.

Reply:
yeah, i think this is one just is what it is...